src/handler-builder.ts

```diff
@@ -35,6 +35,14 @@ import * as remoteConfig from './providers/remoteConfig';
 import * as storage from './providers/storage';
 import * as testLab from './providers/testLab';
 
+/**
+ * The `HandlerBuilder` class facilitates the writing of functions by producers
+ * of Firebase Extensions and developers that want to use gcloud CLI or Cloud Console
+ * to deploy their functions.
+ *
+ * **Do not use `HandlerBuilder` when writing normal functions for deployment via
+ * Cloud Functions for Firebase.** For normal purposes, use `FunctionBuilder`.
```

Comment:
Do you want to link to this `FunctionBuilder` content?

Reply:
Good suggestion, given the potential for confusion here.

Same for your other edits. However, I'm having issues with npm run format after making them, and would rather just merge this with your approval and make the edits. Later.  SG?

I'm honestly unclear about how the formatting is affected, and have been finding npm run format to be very capricious on a day-by-day basis :)

Reply:
yep, that SG.

Reply:
Actually, bizarrely, I tried one more time to make those edits, and now format likes them :)